### ISSUE TITLE
Confirmation of groups deleted

### ISSUE THREAD

**Issue by startmyquest** [bomberstudios/Cleanup-Useless-Groups#9]
A small notification message after plugin has run with how many groups got deleted would be great.

**Comment by pietrogregorini** [comment]
I agree, since it doesn't give any feedbacks if it worked or not. It could be a simple notification like some plugins do (see attachment). Another nice plus would be the number of removed groups each time the plugin is run.

<img width="251" alt="Schermata 2019-07-06 alle 02 50 11" src="https://user-images.githubusercontent.com/3169323/60749350-f4a7e500-9f98-11e9-8611-144ceeaf476c.png">
